server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -41,6 +41,14 @@ func isFinalEvent(obe *pepb.OrderedBuildEvent) bool {
 	return false
 }
 
+func isWorkspaceStatusEvent(bazelBuildEvent build_event_stream.BuildEvent) bool {
+	switch bazelBuildEvent.Payload.(type) {
+	case *build_event_stream.BuildEvent_WorkspaceStatus:
+		return true
+	}
+	return false
```

Comment:
Nit: if you only have one thing in the switch can you just do it like a cast?

<totally untested code>
if _, ok := bazelBuildEvent.Payload.(*build_event_stream.BuildEvent_WorkspaceStatus); ok {
  return true
}
return false

Reply:
Just followed the same pattern as isFinalEvent which switches on one thing immediately above this. 

I could update both, but just going to leave as is because is tried and tested - and will probably go away in the not too distant future.